src/java/org/apache/cassandra/db/RangeTombstoneList.java

```diff
@@ -146,11 +153,16 @@ public void add(RangeTombstone tombstone)
      * This method will be faster if the new tombstone sort after all the currently existing ones (this is a common use case),
      * but it doesn't assume it.
      */
-    public void add(ClusteringBound<?> start, ClusteringBound<?> end, long markedAt, int delTime)
+    public void add(ClusteringBound<?> start, ClusteringBound<?> end, long markedAt, long delTime)
+    {
+        add(start, end, markedAt, CassandraUInt.fromLong(delTime));
+    }
+
+    public void add(ClusteringBound<?> start, ClusteringBound<?> end, long markedAt, int delTimeUnsignedInteger)
```

Comment:
Can be `private`, helping to hide the uint signatures to the rest of the codebase.

Reply:
Let's fold this comment into the previous one.